daphne/src/metrics.rs

```diff
@@ -94,18 +91,16 @@ impl ContextualizedDaphneMetrics<'_> {
             .inc_by(val);
     }
 
-    pub fn agg_job_inc(&self) {
-        self.metrics
-            .aggregation_job_gauge
-            .with_label_values(&[self.host])
-            .inc();
-    }
+    pub fn agg_job_inc(&self, status: DaphneAggregationJobStatus) {
```

Comment:
This works, but to avoid defining a new type (`DaphneAggregationJobStatus`) we could split this into two methods, one for each status: `agg_job_started_inc()` and `agg_job_finished_inc()`. I guess the benefit of defining an enum is that we may eventually define more intermediate statuses?